priceupdater/priceupdater.go

```diff
@@ -173,6 +173,9 @@ func (p *PriceUpdater) UpdatePrices(ctx context.Context) {
 			tokenPrice, err = p.getTokenPriceCoingecko(ctx, token.Addr)
 		case UpdateMethodTypeStatic:
 			tokenPrice = token.StaticValue
+			if tokenPrice == float64(0) {
+				log.Warn("token price is set to 0. Probably StaticValue is not put in the configuration file")
```

Comment:
I suggest adding the token.Symbol in the warning string to help the user determine which token price has the StaticValue at 0.